package.json

```diff
@@ -1,29 +1,43 @@
 {
-    "name": "workshop",
-    "version": "1.1.0",
-    "private": true,
-    "scripts": {
-        "start": "node server",
-        "dev": "nodemon build/server",
-        "build": "node scripts/generateVersionJSON",
-        "postinstall": "npm run build",
-        "test": "npm run lint",
-        "lint": "eslint --ext=.js,.json .",
-        "posttest": "npm audit || true"
-    },
-    "dependencies": {
-        "cors": "2.8.4",
-        "decimal": "0.0.2",
-        "express": "4.16.3",
-        "express-force-ssl": "0.3.2",
-        "helmet": "3.13.0",
-        "morgan": "1.9.0",
-        "request": "2.88.0"
-    },
-    "devDependencies": {
-        "eslint": "5.4.0",
-        "eslint-plugin-json": "1.2.1",
-        "eslint-plugin-node": "7.0.1",
-        "nodemon": "1.18.3"
-    }
+  "name": "workshop",
+  "version": "1.1.0",
+  "private": true,
+  "engines": {
+    "node": ">=8"
+  },
+  "scripts": {
+    "start": "node server",
+    "dev": "nodemon build/server",
+    "build": "node scripts/generateVersionJSON",
+    "postinstall": "npm run build",
+    "test": "npm run lint",
+    "lint": "eslint --ext=.js,.json .",
+    "posttest": "npm audit || true"
+  },
+  "dependencies": {
+    "cors": "2.8.4",
+    "decimal": "0.0.2",
+    "express": "4.16.3",
+    "express-force-ssl": "0.3.2",
+    "helmet": "3.13.0",
+    "morgan": "1.9.0",
+    "request": "2.88.0"
+  },
+  "devDependencies": {
+    "eslint": "5.4.0",
+    "eslint-plugin-json": "1.2.1",
+    "eslint-plugin-node": "7.0.1",
+    "nodemon": "1.18.3"
+  },
+  "description": "Workshop, previously called fhwr-unflattener, serves the Firefox Hardware Report dataset in the format that ensemble-transposer expects.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mozilla/workshop.git"
+  },
+  "keywords": [],
+  "license": "MPL-2.0",
+  "bugs": {
+    "url": "https://github.com/mozilla/workshop/issues"
+  },
+  "homepage": "https://github.com/mozilla/workshop#readme"
 }
```

Comment:
Looks like the indentation was changed somehow.

Reply:
Hhmm... I can change it back to 4. I assume npm did it automatically when I typed <kbd>$ npm init -y</kbd>. Not sure if it'll change it back if we try installing another dependency and it rewrites the file.

Reply:
Which version of NPM are you using? Versions newer than NPM 5 [should preserve indentation](https://github.com/npm/npm/issues/5694).

Reply:
I was using npm v5.6.0 (which is bundled w/ Node v8.11.4 (latest).